src/legacy/index.ts

```diff
@@ -286,7 +286,11 @@ async function buildSubtree(
   nodeId: string,
   eventLoopSpinner: EventLoopSpinner,
   maybeDeduplicationSet: Set<string> | null | false = null, // false = disabled; null = not in deduplication scope yet
```

Comment:
I know it shouldn't be part of this PR, but can we sneak this in?
```suggestion
  maybeDeduplicationSet: Set<string> | null | false = false, // false = disabled; null = not in deduplication scope yet
```
(default to false like we spoke about earlier)